tests/integration/dialog-extension/dialog_events.js

```diff
@@ -86,22 +126,25 @@ asyncTest( "dialog element with no theming", function() {
 			var dialog = $( "#dialog-a" );
 
 			// Assert dialog theme inheritance (issue 1375):
-			ok( dialog.hasClass( "ui-page-theme-a" ), "Expected explicit theme ui-page-theme-a" );
-			ok( $.mobile.pageContainer.hasClass( "ui-overlay-a" ), "Expected default overlay theme ui-overlay-a" );
-			ok( dialog.find( ":jqmData(role=header)" ).hasClass( "ui-bar-inherit" ), "Expected header to inherit from dialog" );
-			ok( dialog.find( ":jqmData(role=content)" ).hasClass( "ui-body-" + $.mobile.page.prototype.options.contentTheme ), "Expect content to inherit from $.mobile.page.prototype.options.contentTheme" );
-			ok( dialog.find( ":jqmData(role=footer)" ).hasClass( "ui-bar-inherit" ), "Expected footer to inherit from dialog" );
+			assert.hasClasses( dialog[ 0 ], "ui-page-theme-a",
+				"Expected explicit theme ui-page-theme-a" );
+			assert.hasClasses( $( "body" )[ 0 ], "ui-overlay-a",
+				"Expected default overlay theme ui-overlay-a" );
+			assert.hasClasses( dialog.find( ":jqmData(role=header)" )[ 0 ], "ui-bar-inherit",
```

Comment:
All of the classes asserts allow jquery objects there is no need to pass the native elements.